Framework/PythonInterface/plugins/algorithms/RefLReduction.py

```diff
@@ -76,7 +77,7 @@ def PyInit(self):
     #pylint: disable=too-many-locals, too-many-branches
     def PyExec(self):
 
-        print '-- > starting new Reflectometer Reduction ...'
+        print('-- > starting new Reflectometer Reduction ...')
```

Comment:
I agree. I know it technically changes behaviour but I'm not sure we want print statements in the algorithms really as they can't be controlled at all.